coap-mbedtls/src/test/java/org/opencoap/transport/mbedtls/MbedtlsNettyTest.java

```diff
@@ -94,6 +108,7 @@ void beforeAll() throws IOException {
                             }
 
                         })
+                        .get("/ctx-hint", __ -> ok("OK!").context(TransportContext.of(DTLS_SESSION_EXPIRATION_HINT, true)).toFuture())
```

Comment:
This filter, could be converted to its own class, under main scope. We should also require that incoming request is nonconfirmable and fail (4.00) if it's not.